include/openthread/tcat.h

```diff
@@ -67,9 +67,10 @@ extern "C" {
 #define OT_TCAT_MAX_SERVICE_NAME_LENGTH \
     15 ///< Maximum string length of a UDP or TCP service name (does not include null char).
 
-#define OT_TCAT_ADVERTISEMENT_MAX_LEN 29 ///< Maximum length of TCAT advertisement.
-#define OT_TCAT_OPCODE 0x2               ///< TCAT Advertisement Operation Code.
-#define OT_TCAT_MAX_VENDORID_SIZE 5      ///< TCAT max size of vendor ID including null as termination.
+#define OT_TCAT_ADVERTISEMENT_MAX_LEN 29       ///< Maximum length of TCAT advertisement.
+#define OT_TCAT_OPCODE 0x2                     ///< TCAT Advertisement Operation Code.
+#define OT_TCAT_MAX_ADVERTISED_VENDORID_SIZE 5 ///< TCAT max size of vendor ID.
```

Comment:
name is a bit confusing in current code! All the variable defs using this constant are called "deviceId" or similar. So at high level there's a thing called "device ID", which has a type, and one type may be a Vendor OUI-24, and another type may be Device Discriminator, etc.   So logically it would be `OT_TCAT_MAX_ADVERTISED_DEVICEID_SIZE` ?
Comment can be used to clarify like in the below:

```suggestion
#define OT_TCAT_MAX_ADVERTISED_DEVICEID_SIZE 5 ///< TCAT max size of any type of advertised Device ID.
```